calico_node/Makefile

```diff
@@ -469,6 +470,8 @@ node-test-containerized: vendor run-etcd-host run-k8s-apiserver
 	-v $(VERSIONS_FILE):/versions.yaml:ro \
 	-e VERSIONS_FILE=/versions.yaml \
 	-e LOCAL_USER_ID=$(LOCAL_USER_ID) \
+	-e DATASTORE_TYPE=$(DATASTORE_TYPE) \
```

Comment:
does it make sense parameterizing this here - since we also pass in the etcd endpoints?  i.e. if you change the type to kubernetes could this work without adding more parameters to this docker command?

Reply:
I think ETCD_ENDPOINTS just gets ignored for KDD. 
I will add the other env var needed for KDD as a parameter and set it to it's default value